bool cmd # Arm=True  Disarm=False
---